cmd/wait-for-github/root.go

```diff
@@ -0,0 +1,134 @@
+// wait-for-github
+// Copyright (C) 2022, Grafana Labs
+
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU Affero General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option) any
+// later version.
+
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more
+// details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/grafana/wait-for-github/internal/github"
+	"github.com/urfave/cli/v2"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var root = &cli.App{
+	Name:  "wait-for-github",
+	Usage: "Wait for a GitHub PR to be merged",
```

Comment:
This should probably not say `PR` or it should mention CI checks since it's the root command